src/settings.py

```diff
@@ -136,14 +136,14 @@ class GlobalSettings:
     grammar_search_true_pos_weight = 10
     grammar_search_false_pos_weight = 1
     grammar_search_bf_weight = 1
-    grammar_search_operator_size_weight = 1e-2
-    grammar_search_pred_complexity_weight = 1
+    grammar_search_operator_size_weight = 0.0
```

Comment:
do we need to pull this into a separate PR? in case it screws something else up?

Reply:
see https://github.com/Learning-and-Intelligent-Systems/predicators/pull/351